mocca-client/src/main/java/com/paypal/mocca/client/MoccaSerializer.java

```diff
@@ -233,15 +235,25 @@ private static boolean isPojo(Type type) {
         return !clazz.isPrimitive();
     }
 
+
+    private static boolean isEnum(Type type) {
+        return  Enum.class.isAssignableFrom((Class<?>) type);
+    }
+
     /*
      * Returns a String containing the specification of a GraphQL variable name and its value,
      * as it is supposed to be written in the request payload
      */
     private String writeRequestVariable(String name, Object value, Type type) {
         final String prefix = name == null ? "" : name + ": ";
-        return type == String.class || type == Character.class || type == OffsetDateTime.class || type == Duration.class || type == UUID.class || type.getTypeName().equals("char") ?
-                prefix + "\\\"" + value.toString() + "\\\"" :
-                prefix + value.toString();
+        if(Enum.class.isAssignableFrom((Class<?>) type)) {
+            return "\\\"" + value + "\\\"";
+        } else {
+            return type == String.class || type == Character.class || type == OffsetDateTime.class || type == Duration.class
+                    || type == UUID.class || type.getTypeName().equals("char") ?
+                    prefix + "\\\"" + value.toString() + "\\\"" :
+                    prefix + value.toString();
+        }
```

Comment:
```suggestion
        return type == String.class || type == Character.class || type == OffsetDateTime.class || type == Duration.class
                || type == UUID.class || type.getTypeName().equals("char") || Enum.class.isAssignableFrom((Class<?>) type) ?
                prefix + "\\\"" + value.toString() + "\\\"" :
                prefix + value.toString();	
```

Reply:
I didn't get this one. Aren't we supposed to remove the prefix if its Enum as that was the concern.